src/plugins/intel_cpu/src/node.cpp

```diff
@@ -152,9 +152,6 @@ Node::Node(const std::shared_ptr<ngraph::Node>& op,
                 str != "cpu:unknown")
                 IE_THROW() << "Unsupported CPU implementation " << str << " for node " << getName();
         }
-        // add default primitive priorities as a fallback for the custom ones
```

Comment:
Could you please ellaborate why you deleted this code?

Reply:
customImplPriorities vector is the implement parsed via Ngraph node RTinfo. Current master code produce the prioritylist by customImplPriorities + default priority list.   If having customImplPriorities, no need to add all supported dnnl::primitive_desc. Just found one.  So in this case, no need to add default priority list as fall back list. If can't found the implement type which customImplPriorities specified. Current code would use the  first_desc  as  fallback

Reply:
customImplPriorities vector is used not only by tests, but also can be set-up by modifing ov::model itself. This is not officialy documented functionality but still used to fix some execution issue on customer side w/o OV recompilation. 
So the idea is to specify some custom priority list (like prioritizing some impls over others), but if, for some specific operations, primitives from custom list cannot be used (like some parameters are not supported) we will fallback on default OV priorities. In other words custom + default lists merge required to relax limitations on custom impls applicability.

Reply:
restore custom + default  back.